src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -6,10 +6,11 @@
 import os
 from datetime import datetime
 from pathlib import Path
-from typing import TYPE_CHECKING, Any, Dict, List, Mapping, Optional, Sequence, Union
+from typing import Any, Dict, List, Mapping, Optional, Sequence, TYPE_CHECKING, Union
 
 import numpy as np
 import xtgeo
+from ert._clib import trans_func  # noqa: no_type_check
```

Comment:
I think perhaps you have `isort` incorrectly set up if it is moving this import for you.